aws/resource_aws_wafv2_rule_group.go

```diff
@@ -86,9 +86,9 @@ func resourceAwsWafv2RuleGroup() *schema.Resource {
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
-									"allow": wafv2EmptySchema(),
-									"block": wafv2EmptySchema(),
-									"count": wafv2EmptySchema(),
+									"allow": wafv2AllowConfigSchema(),
+									"block": wafv2BlockConfigSchema(),
+									"count": wafv2CountConfigSchema(),
```

Comment:
let's also add these newly defined blocks in `wafv2_rule_group.html.markdown`

Reply:
These have been added. Let me know if there's anything I need to fix in the markdown. I tried to follow the existing patterns I could observe there, but its possible I missed something.